languages/java8/template/src/main/java/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,26 @@
+package algorithmia;
+import com.algorithmia.algorithmHandler.*;
+import com.algorithmia.*;
+import java.util.HashMap;
+
+public class Algorithm {
+    String Apply(String input, HashMap<String, String> context) throws Exception{
+            if(context.containsKey("local_file")){
+                return "hello ".concat(input) + " your model file is downloaded here ".concat(context.get("local_file"));
+            }
+        return "hello ".concat(input);
+    }
+    HashMap<String, String> DownloadModel() throws Exception{
```

Comment:
In general, this download function feels strange for a template. Out of the box, it downloads a file that doesn't exist.

Reply:
You're right, I actually have that file & collection path in the `demo` user, I'll remove the `.my` prefix.

Reply:
Is that file present in all enterprise clusters? We will want this to work out-of-box for enterprise customers too.

Reply:
It might make sense to have a demo data file in a users collection, just so we can always show off that api utilization - but that might be scope creep. Would we be fine with using an externally hosted file, like in S3? Or do we want to do away with data API interactions entirely in the advanced template?

Reply:
I always thought that the advanced examples were for internal purposes (for showcasing changes to stakeholders in this project).

We didn't include links to model files in packageset templates, I don't think it makes sense to include them here either. It feels like scope creep.

Reply:
I've adjusted the template to include the advanced functionality in a comment, closing for review.